src/applications/check-in/day-of/pages/TravelVehicle.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+
+import { useFormRouting } from '../../hooks/useFormRouting';
+import Wrapper from '../../components/layout/Wrapper';
+
+const TravelVehicle = props => {
+  const { router } = props;
+  const { t } = useTranslation();
+
+  const { goToNextPage } = useFormRouting(router);
+
+  return (
+    <Wrapper pageTitle="Travel Vehicle">
+      <button
```

Comment:
note to my future self: hopefully we can abstract the content & buttons into a re-usable component like with the confirmation pages.

Reply:
yep, that is what I am doing right now on the content side.